### PR TITLE
Deleted owner in request body

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
Changed owner_type from required to not required in POST petitions. The Tag owner by default is site.

### CODE REVIEW

File: eox_tagging/test/test_viewset.py [hunk 1]
```diff
@@ -75,6 +76,19 @@ def test_create_tag(self):
...
 
+    def test_create_tag_without_owner(self):
+        """"Used to test creating a tag."""
+        data = {
+            "tag_type": "example_tag_2",
+            "tag_value": "example_tag_value",
+            "target_type": "user",
+            "target_id": "user_test",
+            "access": "PRIVATE",
+            "expiration_date": "2020-12-04"
```
Comment (andrey-canon) [inline eox_tagging/test/test_viewset.py:87]: trailing comma

File: eox_tagging/test/test_viewset.py [hunk 2]
```diff
@@ -75,6 +76,19 @@ def test_create_tag(self):
         response = self.client.post("/api/v1/tags/", data, format='json')
         self.assertEqual(response.status_code, 201)
 
+    def test_create_tag_without_owner(self):
+        """"Used to test creating a tag."""
```
Comment (andrey-canon) [inline eox_tagging/test/test_viewset.py:80]: here there is a different condition in the input data and we want a specific response for that data, hence your docstring should show that information, with the current doc-string  my comment would be, why do we need this if the previous test does the same thing ? 
This is just convention, but separate every section with spaces https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Comment (mariajgrimaldi) [reply eox_tagging/test/test_viewset.py:80]: You're right.

File: eox_tagging/test/test_viewset.py [hunk 3]
```diff
@@ -70,11 +71,31 @@ def test_create_tag(self):
...
+        data = {
+            "tag_type": "example_tag_2",
+            "tag_value": "example_tag_value",
+            "target_type": "user",
+            "target_id": "user_test",
+            "access": "PRIVATE",
+            "expiration_date": "2020-12-04",
+        }
+
+        response = self.client.post("/api/v1/tags/", data, format='json')
```
Comment (andrey-canon) [inline eox_tagging/test/test_viewset.py:95]: could you use reverse to generate the url, this should be in the setup https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#reverse

**Pull request merged**